invenio_rdm_records/records/jsonschemas/records/record-v3.0.0.json

```diff
@@ -13,17 +13,16 @@
       "description": "Persistent record identifier (alphanumeric).",
       "type": "string"
     },
-    "pid": {"$ref": "local://records/definitions-v1.0.0.json#/internal-pid"},
-
-    "conceptid": {
-      "description": "Persistent concept record identifier (alphanumeric).",
-      "type": "string"
-    },
-    "conceptpid": {"$ref": "local://records/definitions-v1.0.0.json#/internal-pid"},
+    "pid": {"$ref": "local://records/definitions-v1.1.0.json#/internal-pid"},
 
     "pids": {
-      "additionalProperties": {"$ref": "local://records/definitions-v1.0.0.json#/external-pid"},
-      "description": "Managed persistent identifiers for a record including e.g. OAI-PMH identifier, minted DOIs and more. Managed PIDs are registered in the PIDStore"
+      "type": "object",
```

Comment:
``pids`` actually weren't validated properly before ...... 1) tests were run with v1.0.0 schema instead of v2.0.0 schema, and the v2.0.0 schema had a bug